### ISSUE TITLE
Ordering method is_delivered does not work.

### ISSUE THREAD

**Issue by Embi** [DigitalGlobe/gbdxtools#817]
local environment:
python: 3.6.7
gbdxtools: 0.17.0

```
import gbdxtools
ord = gbdxtools.Ordering()
ord.is_delivered('105001000E383200')
```
fails on:
```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    ord.is_delivered('105001000E383200')
  File "/home/...../python3.6/site-packages/gbdxtools/ordering.py", line 128, in is_delivered
    return state == 'delivered'
NameError: name 'state' is not defined
```

**Comment by drwelby** [comment]
Thanks, this is a bug.

Please use:

```
ord.location('105001000E383200')['acquisitions'][0]['state']
```

to check order status.

**Comment by drwelby** [comment]
Fixed in 0.17.0.post1
